### ISSUE TITLE
Tic tac Toe game

### ISSUE THREAD

**Issue by harshithatnbr** [WH0SUMIT/propack#40]
**Is your feature request related to a problem? Please describe.**
I want to design tic tac Toe game using HTML CSS, javascript.Please assign me this issue . 
Thankyou.

**Comment by WH0SUMIT** [comment]
okay sure @harshithatnbr and do you know how to add projects to the project list ?? if you don't know then you can join our discord server, our team will help you!

**Comment by harshithatnbr** [comment]
Ok I will join. Is the issue assigned to me?

**Comment by WH0SUMIT** [comment]
yes, you can start working on it!  but before merging your first PR please let me know on discord!

**Comment by WH0SUMIT** [comment]
@harshithatnbr Any Updates ??

**Comment by harshithatnbr** [comment]
I will update you by tomorrow sir.

**Comment by WH0SUMIT** [comment]
okay, make sure before making any PR you can join the discord server so we can discuss about how to contribute/add projects in propack !

**Comment by harshithatnbr** [comment]
Ok sir.

**Comment by aditya687** [comment]
Can I work on this issue?

**Comment by WH0SUMIT** [comment]
okay @aditya687 I am assigning this issue to you.

**Comment by WH0SUMIT** [comment]
hey, @aditya687 are you merging this pr under dcp 21 ??

**Comment by aditya687** [comment]
Yes, under dcp2021

**Comment by WH0SUMIT** [comment]
I've send you a form on slack please fill asap ;

**Comment by aditya687** [comment]
Nope I didn't get

**Comment by aditya687** [comment]
Can you check again please else I'll make again?
